src/Psy/ExecutionLoop/ForkingLoop.php

```diff
@@ -166,6 +166,8 @@ private function serializeReturn(array $return)
                 $serializable[$key] = $value;
             } catch (\Exception $e) {
                 // we'll just ignore this one...
+            } catch (\Error $e) {
```

Comment:
Sure. Might as well use `Throwable` here. It should be more future proof. You know, in case PHP 8 decides we should be able to throw more than just Errors and Exceptions :)